src/jobs.rs

```diff
@@ -361,6 +361,18 @@ impl CommitJobInstance {
 
         let mut commits = HashMap::new();
 
+        let mut endoflife_rebase_arg = String::new();
```

Comment:
This is kind of weird rust. The more "correct" way to do it is to make it a non-mutable Option<String>.
Something like 
```
let endoflife_rebase = if let  Some(endoflife_rebase) = &self.endoflife_rebase {
  construct string;
  Some(result)
} else {
  None
}
```